src/Map.cpp

```diff
@@ -151,14 +151,12 @@ bool cMap::UpdatePixel(unsigned int a_X, unsigned int a_Z)
 				ChunkHeight /= 4;
 				while (((--Height) != -1) && IsBlockWater(a_Chunk.GetBlock(RelX, Height, RelZ)))
 				{
-					continue;
```

Comment:
It's better and more explicit without, I think an empty brace block just looks weird.